consumer/commons/src/main/java/org/eclipse/kapua/consumer/commons/xml/ConsumerJAXBContextInitializer.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * Copyright (c) 2017, 2022 Eurotech and/or its affiliates and others
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Eurotech - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.kapua.consumer.commons.xml;
+
+import org.eclipse.kapua.KapuaException;
+import org.eclipse.kapua.commons.util.xml.JAXBContextProvider;
+import org.eclipse.kapua.locator.KapuaLocator;
+
+public class ConsumerJAXBContextInitializer {
+    private void init() {
```

Comment:
This class was introduced in place of the previous ConsumerJAXBContextLoader, with the sole purpose of triggering the class path scan during spring's application context initialization